packages/phone-number-privacy/signer/src/server.ts

```diff
@@ -17,8 +18,14 @@ export enum Endpoints {
   METRICS = '/metrics',
   GET_BLINDED_MESSAGE_PARTIAL_SIG = '/getBlindedMessagePartialSig',
   GET_QUOTA = '/getQuota',
+
+  DISABLE_DOMAIN = '/domain/disable',
+  DOMAIN_SIGN = '/domain/sign/',
+  DOMAIN_STATUS = '/domain/status',
```

Comment:
Overall, I like these paths better than what I suggested in CIP-40. I'll go ahead and amend that CIP to match this. A would suggest `quotaStatus` instead of `status` to clarify that this is not the status endpoint for the service itself.
```suggestion
  DOMAIN_STATUS = '/domain/quotaStatus',
```

Reply:
+1